client/shared/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -169,11 +169,22 @@ trait Client[F[_]] {
     */
   def get[A](s: String)(f: Response[F] => F[A]): F[A]
 
+  @deprecated("use public method with MonadCancelThrow instead", since = "2021-Dec")
+  private[client] def translate[G[_]: Async](
+      fk: F ~> G
+  )(gK: G ~> F)(implicit F: MonadCancelThrow[F]): Client[G] =
+    Client((req: Request[G]) =>
```

Comment:
Can this just delegate to the new method? I think you may have to pass the implicits explicitly and "cast" their types to `MonadCancelThrow` to resolve the ambiguity, but I think it should work.

Reply:
Updated